examples/pipelines/controlnet-interior-design/run_locally.sh

```diff
@@ -0,0 +1,19 @@
+export METADATA='{"run_id":"test_niels", "base_path":"/Users/nielsrogge/Documents/fondant_artifacts"}'
```

Comment:
I'm a bit conflicted about having this merged on the main branch.
Do we want to set an example/way of working on running things locally like this @GeorgesLorre ? 
Maybe leave out your name from it and just keep some defaults